internal/v6provider/resource_load_balancer.go

```diff
@@ -73,68 +93,45 @@ func (r LoadBalancerResource) Schema(
 				Computed: true,
 			},
 			"name": schema.StringAttribute{
-				Computed: true,
-				Optional: true,
+				Required: true,
 			},
-			"resource_type": schema.StringAttribute{
-				Computed: true,
-			},
-			"virtual_machine": schema.ListNestedAttribute{
+			"virtual_machine_ids": schema.SetAttribute{
 				Optional: true,
-				Computed: true,
-				Validators: []validator.List{
-					listvalidator.ConflictsWith(
-						path.MatchRoot("tag"),
-						path.MatchRoot("virtual_machine_group"),
+				Validators: []validator.Set{
+					setvalidator.ConflictsWith(
+						path.MatchRoot("tag_ids"),
+						path.MatchRoot("virtual_machine_group_ids"),
 					),
 				},
-				NestedObject: schema.NestedAttributeObject{
-					Attributes: map[string]schema.Attribute{
-						"id": schema.StringAttribute{
-							Required: true,
-						},
-					},
-				},
+				ElementType: types.StringType,
 			},
-			"virtual_machine_group": schema.ListNestedAttribute{
+			"virtual_machine_group_ids": schema.SetAttribute{
 				Optional: true,
-				Computed: true,
-				Validators: []validator.List{
-					listvalidator.ConflictsWith(
-						path.MatchRoot("tag"),
-						path.MatchRoot("virtual_machine"),
+				Validators: []validator.Set{
+					setvalidator.ConflictsWith(
+						path.MatchRoot("tag_ids"),
+						path.MatchRoot("virtual_machine_ids"),
 					),
 				},
-				NestedObject: schema.NestedAttributeObject{
-					Attributes: map[string]schema.Attribute{
-						"id": schema.StringAttribute{
-							Required: true,
-						},
-					},
-				},
+				ElementType: types.StringType,
 			},
-			"tag": schema.ListNestedAttribute{
+			"tag_ids": schema.SetAttribute{
 				Optional: true,
-				Computed: true,
-				Validators: []validator.List{
-					listvalidator.ConflictsWith(
-						path.MatchRoot("virtual_machine"),
-						path.MatchRoot("virtual_machine_group"),
+				Validators: []validator.Set{
+					setvalidator.ConflictsWith(
+						path.MatchRoot("virtual_machine_ids"),
+						path.MatchRoot("virtual_machine_group_ids"),
 					),
 				},
-				NestedObject: schema.NestedAttributeObject{
-					Attributes: map[string]schema.Attribute{
-						"id": schema.StringAttribute{
-							Required: true,
-						},
-					},
-				},
+				ElementType: types.StringType,
 			},
```

Comment:
I know we did remove it, but I believe we should temporarily add back `Computed: true` to the resource attributes. A fix to the API to reject create/update requests with invalid resource IDs is coming, but it's not live yet.

Hence while the list of IDs returned might be different than those Terraform sent, we should use `Computed: true` for now, annoying as it may be.